Nim/Nim/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿// See https://aka.ms/new-console-template for more information
+// https://github.com/forsbergsskola-se/gp24-102-csharp-basics-Hunzlah
+
+bool isPlayerTurn = true;
+int numberOfMatches = 24;
+
+Console.WriteLine("Output: Welcome to Nim!");
+DisplayRemainingMatches();
+Console.WriteLine("Output: How many matches do you want to draw?");
+
+while (numberOfMatches > 0)
+{
+    int playerInput;
+    if (isPlayerTurn)
+    {
+        Console.Write("Input: ");
+        playerInput = Convert.ToInt32(Console.ReadLine());
+
+        if (playerInput < 1 || playerInput > 3)
+        {
+            Console.WriteLine("Invalid input.");
+            continue;
+        }
+        else
+        {
+            isPlayerTurn = !isPlayerTurn;
+        }
+    }
+    else
+    {
+        playerInput = new Random().Next(1,4);
+        isPlayerTurn = !isPlayerTurn;
+    }
```

Comment:
You could simplify this a bit:

```c#
if(isPlayerTurn)
{
    // input handling
    if(invalid) continue;
    // remove the else
}
else
{
   // ai input
}
isPlayerTurn = !isPlayerTurn; // new. This was removed from line 26 and 32 and instead moved after line 33. Because it is called either way :)